### PR TITLE
[RFC] Clean-up before restarting single-includes test

### PR DESCRIPTION
**Pull request by ZyX-I**
May fix issue with failures.

### CODE REVIEW

File: ci/run_lint.sh [hunk 1]
```diff
@@ -12,8 +12,17 @@ enter_suite 'lint'
 
 set -x
 
+csi_clean() {
+  rm "${BUILD_DIR}"/bin/test-includes-*
```
Comment (jamessan) [inline ci/run_lint.sh]: Need `rm -f ...` in case no files match the glob, to avoid failing the script due to `set -e`.

**Comment by jamessan** [comment]
The latest run exhausted retries, but result in a "green" status.  Is that intended?

**Comment by justinmk** [comment]
Is this ready? 

macOS build reported a crash:

```
make: *** [test49.out] Error 139
++ reset

        H        H        H        H        H        H        H        H        H
c>
++ asan_check /Users/travis/build/neovim/neovim/build/log
++ check_logs /Users/travis/build/neovim/neovim/build/log '*san.*'
+++ find /Users/travis/build/neovim/neovim/build/log -type f -name '*san.*'
++ local err=
+++ find /Users/travis/build/neovim/neovim/build/log -type f -name '*san.*' -size +0
++ [[ -n '' ]]
++ valgrind_check /Users/travis/build/neovim/neovim/build/log
++ check_logs /Users/travis/build/neovim/neovim/build/log 'valgrind-*'
+++ find /Users/travis/build/neovim/neovim/build/log -type f -name 'valgrind-*'
++ local err=
+++ find /Users/travis/build/neovim/neovim/build/log -type f -name 'valgrind-*' -size +0
++ [[ -n '' ]]
++ check_core_dumps
++ local del=
++ test '' = --delete
++ local app=/Users/travis/build/neovim/neovim/build/bin/nvim
++ [[ osx == osx ]]
+++ find /cores/ -type f -print
++ local cores=/cores//core.22705
++ '[' -z /cores//core.22705 ']'
++ local core
++ for core in '$cores'
++ test '' = 1
++ print_core /Users/travis/build/neovim/neovim/build/bin/nvim /cores//core.22705
++ local app=/Users/travis/build/neovim/neovim/build/bin/nvim
++ local core=/cores//core.22705
++ test /Users/travis/build/neovim/neovim/build/bin/nvim = quiet
++ echo '======= Core file /cores//core.22705 ======='
======= Core file /cores//core.22705 =======
++ [[ osx == osx ]]
++ lldb -Q -o 'bt all' -f /Users/travis/build/neovim/neovim/build/bin/nvim -c /cores//core.22705
(lldb) target create "/Users/travis/build/neovim/neovim/build/bin/nvim" --core "/cores//core.22705"
warning: (x86_64) /cores/core.22705 load command 107 LC_SEGMENT_64 has a fileoff + filesize (0x2b968000) that extends beyond the end of the file (0x2b967000), the segment will be truncated to match
warning: (x86_64) /cores/core.22705 load command 108 LC_SEGMENT_64 has a fileoff (0x2b968000) that extends beyond the end of the file (0x2b967000), ignoring this section
Core file '/cores/core.22705' (x86_64) was loaded.
(lldb) bt all
warning: could not load any Objective-C class information. This will significantly reduce the quality of type information available.
* thread #1: tid = 0x0000, 0x00007fff8abb4f8a libsystem_kernel.dylib`__read_nocancel + 10, stop reason = signal SIGSTOP
  * frame #0: 0x00007fff8abb4f8a libsystem_kernel.dylib`__read_nocancel + 10
    frame #1: 0x00007fff9b762155 libsystem_c.dylib`_sread + 16
    frame #2: 0x00007fff9b761769 libsystem_c.dylib`__srefill1 + 24
    frame #3: 0x00007fff9b75aa60 libsystem_c.dylib`fgets + 106
    frame #4: 0x000000010d8fd431 nvim`get_one_sourceline(sp=0x00007fff523b1e00) + 145 at ex_cmds2.c:3162
    frame #5: 0x000000010d8fd0df nvim`getsourceline(c=0, cookie=0x00007fff523b1e00, indent=0) + 185 at ex_cmds2.c:3070
    frame #6: 0x000000010d8fc6b5 nvim`do_source(fname="test49.vim", check_other=0, is_vimrc=0) + 1136 at ex_cmds2.c:2846
    frame #7: 0x000000010d8fc195 nvim`cmd_source(fname="test49.vim", eap=0x00007fff523b1f90) + 183 at ex_cmds2.c:2677
    frame #8: 0x000000010d8fc0dc nvim`ex_source(eap=0x00007fff523b1f90) + 34 at ex_cmds2.c:2658
    frame #9: 0x000000010d9039de nvim`do_one_cmd(cmdlinep=0x00007++ test /Users/travis/build/neovim/neovim/build/bin/nvim = quiet
++ exit 1
```

**Comment by ZyX-I** [comment]
Should be ready. It appears that `ag -u` did not find that because I was writing `temp` in place of `tmp`.

**Pull request merged**